CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+Release NEXT:
+- Update to latest KMP Crypto
+  - Introduces correct mulitbase encoding
+  - EC Point Compression
+  - **THIS IS A BREAKING CHANGE WRT. SOME ASPECTS OF SERIALIZATION**
```

Comment:
We should at least document this breaking change here (essentially plain byte encoding vs var int encoding, right?)

Reply:
you mean more elaboration?

Reply:
better now?